source/shared/cpp/ObjectModel/CollectionTypeElement.cpp

```diff
@@ -46,7 +46,7 @@ void CollectionTypeElement::SetPadding(const bool value)
 void CollectionTypeElement::ConfigPadding(const ParseContext& context)
 {
     // We set padding when child's style is set explicitly (not None) and is different than the parental style or if the style is emphasis or if background image is not present
-    const bool padding = ( GetBackgroundImage() != NULL || ((GetStyle() != ContainerStyle::None) && (context.GetParentalContainerStyle()!= GetStyle())) || GetStyle() == ContainerStyle::Emphasis);
+    const bool padding = ((GetBackgroundImage() == nullptr && GetStyle() != ContainerStyle::None && (context.GetParentalContainerStyle()!= GetStyle())) || GetStyle() == ContainerStyle::Emphasis);
```

Comment:
what is this change intended for?

Reply:
The condition for apply padding was not fixed correctly the last time and was creating problem when background image was present. So the conditions are checked correctly this time

Reply:
This change is applied in the shared cpp.... I think we should discuss the same with the Mac and MS team as well.

Reply:
I will cross check once again. The last time also the conditions were modified by me based on Apply Padding from HTML Renderer